ci/test/pipeline.template.yml

```diff
@@ -457,6 +517,17 @@ steps:
         agents:
           queue: hetzner-aarch64-4cpu-8gb
 
+      - id: pg-rtr-old-syntax
+        label: Postgres RTR tests (before source versioning)
+        depends_on: build-aarch64
+        timeout_in_minutes: 30
+        inputs: [test/pg-rtr-old-syntax]
+        plugins:
+          - ./ci/plugins/mzcompose:
+              composition: pg-rtr-old-syntax
+        agents:
+          queue: hetzner-aarch64-4cpu-8gb
+
```

Comment:
Can we move all the old syntax tests to nightly? I'm guessing they have lower risk to change and the new syntax tests cover a lot of the same surface area in the code.

Reply:
Yeah, makes sense

Reply:
✅